rust/core-lib/src/tabs.rs

```diff
@@ -689,7 +689,7 @@ impl CoreState {
                     .find(|v| v.borrow().get_buffer_id() == buffer_id)
                     .map(|v| v.borrow().get_view_id())
                     .unwrap();
-                self.make_context(view_id).unwrap().reload(text);
+                self.make_context(view_id).unwrap().reload_tail(text);
```

Comment:
We want to keep the old reload behavior for files that are not being tailed, no?

Reply:
Good point. But the method containing this `handle_open_file_fs_event` is tagged as notify. Besides, we monitor four notify events: NoticeWrite, Write, Chmod and Create. `reload_tail` will handle all four correctly unless Im missing a test case.

Reply:
I think there's a misunderstanding, here. I'm going to explain the 'notify' tag in my top level comment; but in this function what we should be doing, I think, is checking if the file is marked as being in 'tail mode', and if it is we call `reload_tail()` instead of `reload()`. We don't want to change anything about the existing reload behaviour, which should still continue to work.